vic/vic_run/src/CalcBlowingSnow.c

```diff
@@ -363,24 +363,29 @@ polint(double  xa[],
  *****************************************************************************/
 double
 trapzd(double (*funcd)(),
-       double es,
-       double Wind,
-       double AirDens,
-       double ZO,
-       double EactAir,
-       double F,
-       double hsalt,
-       double phi_r,
-       double ushear,
-       double Zrh,
-       double a,
-       double b,
-       int    n)
+       double   es,
+       double   Wind,
+       double   AirDens,
+       double   ZO,
+       double   EactAir,
+       double   F,
+       double   hsalt,
+       double   phi_r,
+       double   ushear,
+       double   Zrh,
+       double   a,
+       double   b,
+       int      n)
 {
     double        x, tnm, sum, del;
     static double s;
+
     int           it, j;
 
+    // TODO: remove use of static variables (see GH #735), for now:
+    // make static variables thread safe
+    #pragma omp threadprivate(s)
```

Comment:
I put it here because I thought it was cleaner to collect all the private variables in the same place. This function only had one so that's why it stands alone. I'll see if I can move it to make it clearer.